doc/releases/release-notes-3.6.rst

```diff
@@ -29,6 +29,10 @@ Architectures
 
 * ARC
 
+ * Extended support for ARCv3 features: enable HW prefetch and shared cache;
```

Comment:
Improvements for ARCv3 processors (HS5x & HS6x): enable hardware prefetcher and shared cluster cache (SCM - Shared Cluster Memory).